docker-compose.yml

```diff
@@ -1,17 +1,38 @@
-version: '3.1'
+version: '3.4'
 
 services:
+    frontend:
+        build:
+            context: .
+            dockerfile: ./Dockerfile.frontend.dev
+        ports:
+            - 5001:5001
```

Comment:
TODO: maybe not necessary, depending on how frontend content is served locally?

Reply:
Ah I see. There's a flag (`onionpush`) that serves the content from `public/` directory on 5001.

Reply:
That's true.

Marking this as resolved since the frontend container is no longer needed.